### PR TITLE
March 2023 newsletter

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
Looks good, and straightforward 👍

File: content/posts/newsletter-march-2023.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  with a number of examples and tips to help campaigns perform their best.
+* We made some improvements to the landing pages that compare us against
+  [Carbon]({filename}../pages/vs-carbon-ads.md) and [Google]({filename}../pages/vs-google.md).
+* We made a number of improvements to our open source ad server
+  which aim to help the folks who use it for their own purposes.
+  We'd received some helpful feedback and wanted to show the community some love.
+* The improvements we made in [February]({filename}../posts/newsletter-february-2023.md#new-features)
+  to pacing and prioritization were rolled out to (almost) everybody.
+* We are making a lot of improvements to our sales process.
+  We have another announcement coming but we are onboarding somebody to help with sales for the next few months.
```
[the remark this answers is not in the document]
Comment (davidfischer) [reply content/posts/newsletter-march-2023.md:26]: Should we remove this section?

[the remark this answers is not in the document]
Comment (ericholscher) [reply content/posts/newsletter-march-2023.md:26]: I'm not sure? I don't really know what it's referring to, so not sure if it's appropriate or not!

[the remark this answers is not in the document]
Comment (davidfischer) [reply content/posts/newsletter-march-2023.md]: I'll make it more clear. This is about our new sales rep.

**Pull request merged**